### PR TITLE
Add RzIL info to rz-asm and La

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Adds info about which plugin has RzIL support or not.

### CODE REVIEW

File: librz/core/casm.c [hunk 1]
```diff
@@ -99,7 +111,7 @@ RZ_API RzCmdStatus rz_core_asm_plugin_print(RzCore *core, RzAsmPlugin *ap, const
 			strcat(bits, "32 ");
 		}
 		if (ap->bits & 64) {
-			strcat(bits, "64 ");
+			strcat(bits, "64");
```
Comment (XVilka) [inline librz/core/casm.c:114]: Why this change? Other bitness values have space as well.

**Review by XVilka — CHANGES_REQUESTED** [review]
The rest LGTM

File: librz/core/casm.c (hunk 1 above)
Comment (XVilka) [reply librz/core/casm.c:114]: Nevermind, I see that is the last value, so the space is unnecessary.

**Review by XVilka — APPROVED** [review]

File: librz/core/casm.c (hunk 1 above)
Comment (wargio) [reply librz/core/casm.c:114]: because is always the last one and doesn't need the extra space.
